### DISCUSSION TITLE
Adding more chromium flags?

### DISCUSSION THREAD

**Discussion by huyz** [ArchiveBox/ArchiveBox#1480] [Q&A] [answered]
Is there a way to pass more flags to the Chromium binary that is launched by ArchiveBox?

I need to add `--load-extension=` to load an unpacked extension.  I tried to do it interactively through the usual browser settings but Chromium crashes as soon as I click the `Load unpacked` button.  So I'm trying to do it through the command line now.

**Comment by pirate** [comment, marked as answer]
`CHROME_EXTRA_ARGS` is available in >= [`0.8.0-rc1`](https://github.com/ArchiveBox/ArchiveBox/releases/tag/v0.8.0-rc) (aka `dev` right now): https://github.com/ArchiveBox/ArchiveBox/blob/dev/archivebox/config.py#L156C10-L156C27

**Reply by huyz** [reply]
Are there instructions on how to package up my own Docker image from the latest on `dev` and run that?

**Reply by pirate** [reply]
I would not recommend building on `dev` just yet, I'm still changing it heavily. There are a lot of competing priorities with scaling my ArchiveBox consulting business but getting v0.8 ready is high on my list still!

There are some relevant docs here though if you're ok dealing with pre-release instability/frequent changes: 
- https://github.com/ArchiveBox/ArchiveBox#install-and-run-a-specific-github-branch
- https://github.com/ArchiveBox/ArchiveBox#setup-the-dev-environment
- https://github.com/ArchiveBox/ArchiveBox#build-the-docs-pip-package-and-docker-image

**Reply by huyz** [reply]
Looks like the pre-release contains the `CHROME_EXTRA_ARGS` so I can try that: https://github.com/ArchiveBox/ArchiveBox/releases/tag/v0.8.0-rc
